distrepos/mirror_run.py

```diff
@@ -0,0 +1,115 @@
+"""
+This module contains the functions for populating mirrors for a single tag.
+The main entry point is update_mirrors_for_tag(); other functions are helpers.
+"""
+
+import logging
+from distrepos.params import Options, Tag
+from distrepos.tag_run import update_release_repos
+import typing as t
+import socket
+import string
+import os
+import requests
+from datetime import datetime, timedelta
+from pathlib import Path
+
+_log = logging.getLogger(__name__)
+
+def get_baseline_urls() -> t.List[str]:
+    """
+    Get the 
+    """
+    timeout = 5
+    socket.setdefaulttimeout(timeout)
+    if 'repo-itb' in socket.gethostname():
```

Comment:
```suggestion
    fqdn = socket.getfqdn()
    if "osgdev" in fqdn or "osg-dev" in fqdn:
```
I know this was from the original mirror script -- it was broken there too. More precisely, it didn't work with Kubernetes, where `socket.gethostname()` just returns `osg-repo-0`. However, the FQDN does appear to have the namespace in it, which you could use instead. (Tiger and Tempest use `osgdev` as the dev namespace, River uses `osg-dev`)